ui/src/pages/Profile/index.tsx

```diff
@@ -0,0 +1,18 @@
+import {ProfileCard} from '../../components/ProfileCard';
+import {Sidebar} from '../../components/Sidebar/Sidebar';
+import {TopBar} from '../../components/TopBar';
+import styles from './styles.module.css';
+
+export const Profile: React.FC = () => {
+  return (
+    <div className={styles.container}>
+      <div className={styles.NavContainer}>
+        <Sidebar />
+      </div>
+      <div className={styles.mainContent}>
+        <TopBar />
```

Comment:
These are duplicated (same in the `Home` component I believe).

It doesn't have to be in this PR, create a separate issue to keep the common part in a common component (maybe `Layout.tsx`)

Reply:
Created an issue for this.